### ISSUE TITLE
Categories list with supported & required parameters

### ISSUE THREAD

**Issue by Maxmilianos** [allegro/allegro-api#9750]
Is there somewhere to download allegro categories including supported and required parameters?
Ideal in csv/xlsx

**Comment by AureliuszBrussy** [comment]
You can fetch the category tree with [GET /sale/categories](https://developer.allegro.pl/documentation#operation/getCategoriesUsingGET).

You retrieve the parameters for each category with [GET /sale/categories/{categoryId}/parameters](https://developer.allegro.pl/documentation#operation/getFlatParametersUsingGET_2). 

 I don't have any resource to download the entire tree to an .xml or .csv file.

**Comment by Maxmilianos** [comment]
Yes, I know about this. Unfortunately, I was looking for a full list of categories including their parameters.
I've been trying to get it this way since yesterday, but I'm not even halfway through.

Thank you for your reply.
